src/heatmap/HeatMapImageRenderer.ts

```diff
@@ -18,6 +20,8 @@ function ensureHex(color: string) {
   return `#${toHex(rgb.r)}${toHex(rgb.g)}${toHex(rgb.b)}`;
 }
 
+const TOO_LONG_URL = 4096;
```

Comment:
Name is a little bit misleading. 4096 is not too long - 4097 would be. I would call it something TDP_URL_LENGTH_LIMIT or MAX_URL_LENGTH

Also your check in line 145 says `>` and not `>=`

Reply:
We extracted this constant to phovea_core with PR phovea/phovea_core#150